docs/project/library-servicing.md

```diff
@@ -8,33 +8,7 @@ If a library's source project sets `<IsPackable>true</IsPackable>` a package is
-
-The reason we need to increment the assembly version for things running on .NET Framework is because of the way binding works there. If there are two assemblies with the same assembly version the loader will essentially pick the first one it finds and use that version so applications don't have full control over using the later build with a particular fix included. This is worse if someone puts the older assembly in the GAC as the GAC will always win for matching assembly versions so an application couldn't load the newer one because it has the same assembly version.
-
-If this library ships both inbox on a platform and in its own library package then we need to keep the reference assembly version pinned to the same version that was shipped inbox on. In those cases we generally need to condition the AssemblyVersion in the library `ref` project like such:
-
-```
-    <!-- Must match version supported by frameworks which support 4.0.* inbox.
-         Can be removed when API is added and this assembly is versioned to 4.1.* -->
-    <AssemblyVersion Condition="!$(TargetFramework.StartsWith('net4'))' != 'true'">4.0.3.0</AssemblyVersion>
-```
-Where the `AssemblyVersion` is set to the old version before updating. To determine if the library ships inbox you can look at the list in [NetCoreAppLibrary.props](https://github.com/dotnet/runtime/blob/95147163dac477da5177f5c5402ae9b93feb5c89/src/libraries/NetCoreAppLibrary.props#L1).
-
-If the library is part of a Aspnetcore or .NET targeting pack then we cannot increment the assembly version. For Aspnetcore, You can examine the ```<IsAspNetCoreApp>true</IsAspNetCoreApp>``` property in the library`s ```Directory.Build.props```
-For .Net you can examine the list in ```NetCoreAppLibrary.props```
-
-If the library is part of a targeting pack and also contains an asset applicable to the .NET Framework then we will increment the assembly version for that asset.
-eg.
-```
-<AssemblyVersion Condition="$(TargetFramework.StartsWith('net4'))">6.0.0.1</AssemblyVersion>
-```
+When you make a change to a library & ship it during the servicing release, the `ServicingVersion` must be bumped. This property is found in the library's source project. It's also possible that the property is not in that file, in which case you'll need to add it to the library's source project and set it to 1. If the property is already present in your library's source project, just increment the servicing version by 1.
```

Comment:
> ## Check if a package is generated

In the section above, can you please update the content to express that packages aren't generated during servicing by default and `<GeneratePackageOnBuild>true</GeneratePackageOnBuild>` needs to be set in the source project file?